libs/cdk/utils/directives/auto-complete/auto-complete.directive.ts

```diff
@@ -72,6 +75,12 @@ export class AutoCompleteDirective {
          */
         this._zone.runOutsideAngular(() => {
             const keyupEvent = fromEvent<KeyboardEvent>(this._elementRef.nativeElement, 'keyup');
+            const compositionStartEvent = fromEvent<CompositionEvent>(
+                this._elementRef.nativeElement,
+                'compositionstart'
```

Comment:
![Screenshot 2024-09-03 at 1 47 09 PM](https://github.com/user-attachments/assets/c93f0b55-217c-4bf6-8582-3a7a2d5ed6ba)
![Screenshot 2024-09-03 at 1 47 00 PM](https://github.com/user-attachments/assets/c2da62fc-7ea2-4928-a143-91a5d26389e3)

'compositionstart' and 'compositionend' are experimental technology and not supported in all browsers:
https://developer.mozilla.org/en-US/docs/Web/API/EditContext/compositionstart_event

Reply:
Hi @InnaAtanasova, I found this [MDN article on the `compositionstart` event](https://developer.mozilla.org/en-US/docs/Web/API/Element/compositionstart_event), which mentions that it is supported by all major browsers. I also tested it on Safari, Edge, Firefox, and Chrome, and it worked without any issues.